trp_project_scrum/security/ir_rules_collaborators.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="False">
+        <record id="project_rule_memb" model="ir.rule">
+            <field name="name">Show only collaborator Project</field>
+            <field name="model_id" ref="project.model_project_project"/>
+            <field name="domain_force">[('members', 'in', [user.id] ), ('use_scrum', '=', True)]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="project_task_rule_memb" model="ir.rule">
+            <field name="name">Show Only collaborator Task</field>
+            <field name="model_id" ref="project.model_project_task"/>
+            <field name="domain_force">[
+                    ('user_id', '=', user.id), 
+                    ('project_id.members', 'in', [user.id] ),
+                    ('use_scrum', '=', True)]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="True" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+
+        <record id="sprint_rule_memb" model="ir.rule">
+            <field name="name">Sprint</field>
+            <field name="model_id" ref="project_scrum.model_project_scrum_sprint"/>
+            <field name="domain_force">[('project_id.members', 'in', [user.id] )]</field>
```

Comment:
I lost some time here , i read the cook book but no explanation , why is the domain reversed here?
It is very counter intuitive. A many2many  'in' a id  instead of vice versa  
this reversal isn't in https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_rule.py#L48

i see we get "user" here:
https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_rule.py#L48

but other than normalize_domain which i know what it does i see no reason for this nomenclature.